### PR TITLE
Added CSS dominant-baseline property

### PR DESCRIPTION
**Pull request by meyerweb**
### Description

Adds documentation of the CSS property `dominant-baseline`, which replicates an SVG attribute.

### Motivation

The page currently doesn’t exist, and needs to be created (see https://github.com/mdn/content/issues/34763).

### CODE REVIEW

File: files/en-us/web/css/dominant-baseline/index.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  <text x="250" y="20">mathematical</text>
+  <text x="250" y="60">middle</text>
+  <text x="250" y="100">text-bottom</text>
+  <text x="250" y="140">text-top</text>
+  <path
+    d="M   0,20 l 400,0
+       m -400,40 l 400,0
+       m -400,40 l 400,0
+       m -400,40 l 400,0"
+    stroke="grey" />
```
Comment (estelle) [inline files/en-us/web/css/dominant-baseline/index.md:105]:
```suggestion
    stroke="grey" />
    
  <text x="0" y="20" fill="red">auto</text>
  <text x="0" y="60" fill="red">auto</text>
  <text x="0" y="100" fill="red">auto</text>
  <text x="0" y="140" fill="red">auto</text>
```
optionally, you can add these so users can see where the text lands by default
Also, confirming that text-top has no effect in this case?

**Review by estelle — COMMENTED** [review]
I don't understand one of the sentences, which is repeated a few times.

File: files/en-us/web/css/dominant-baseline/index.md (hunk 1 above)
Comment (meyerweb) [reply files/en-us/web/css/dominant-baseline/index.md:105]: text-top and -bottom seem to be ignored in all browsers.

**Review by estelle — APPROVED** [review]
thank you!

will check once live if still redirecting.

**Pull request merged**